caddy/https/user.go

```diff
@@ -104,7 +104,7 @@ func saveUser(user User) error {
 // instead. It does NOT prompt the user.
 func newUser(email string) (User, error) {
 	user := User{Email: email}
-	privateKey, err := rsa.GenerateKey(rand.Reader, rsaKeySizeToUse)
+	privateKey, err := rsa.GenerateKey(rand.Reader, KeySize)
```

Comment:
You are right, but at the moment we only support RSA Private Keys.

I am working on it